cypress/e2e/specialFeatures/liteSiteWeight/helpers/interceptGetRequests.js

```diff
@@ -0,0 +1,7 @@
+export default allRequests => {
+  cy.intercept('GET', '**', request => {
+    request.continue(async response => {
```

Comment:
Do we need async here?
```suggestion
    request.continue(response => {
```